tests/sentry/api/endpoints/notifications/test_notification_actions_index.py

```diff
@@ -36,18 +50,30 @@ def setUp(self):
             self.create_project(name="greenpath", organization=self.organization),
             self.create_project(name="dirtmouth", organization=self.organization),
         ]
-        self.base_data = {
+        self.base_data: MutableMapping[str, Any] = {
             "serviceType": "email",
             "triggerType": "audit-log",
             "targetType": "specific",
             "targetDisplay": "@hollowknight",
             "targetIdentifier": "THK",
         }
-        self.mock_register = lambda data: NotificationAction.register_action(
-            trigger_type=ActionTrigger.get_value(data["triggerType"]),
-            service_type=ActionService.get_value(data["serviceType"]),
-            target_type=ActionTarget.get_value(data["targetType"]),
-        )
+
+        def mock_register(
+            data: MutableMapping[str, Any]
+        ) -> Callable[[type[ActionRegistrationT]], type[ActionRegistrationT]]:
+            triggerType = ActionTrigger.get_value(data["triggerType"])
+            serviceType = ActionService.get_value(data["serviceType"])
+            targetType = ActionTarget.get_value(data["targetType"])
+
+            assert triggerType is not None, "triggerType must exist"
+            assert serviceType is not None, "serviceType must exist"
+            assert targetType is not None, "targetType must exist"
+
+            return NotificationAction.register_action(
+                trigger_type=triggerType, service_type=serviceType, target_type=targetType
+            )
+
+        self.mock_register = mock_register
```

Comment:
it's a bit weird that this is a function that's assigned to `self` -- I think it can just be a free function at the module scope?

also there's some camelCase in here you'll want to snake_case

Reply:
the python variables should be snake case -- the dictionary values can be camel case though

Reply:
ohhhhh mb,  thats a dentge moment from me